code/Model/EditableFormField/EditableTextField.php

```diff
@@ -101,6 +101,21 @@ public function getCMSFields()
         return parent::getCMSFields();
     }
 
+    /**
+     * @return ValidationResult
+     */
+    public function validate()
+    {
+        $result = parent::validate();
+
+        if ($this->MinLength > $this->MaxLength) {
+            $result->addError("Minimum length should be less than the Maximum length.");
```

Comment:
Thanks for the contribution!
Could you please make this string [translatable](https://docs.silverstripe.org/en/4/developer_guides/i18n/#translating-text)?

Reply:
Awesome work. Unfortunately now the "linter" is unhappy 😢 : 

```
 112 | ERROR | [x] Expected 0 spaces before closing bracket; 1 found
```

The linter just is an automated tool that checks the code meets certain style requirements. In your case there's an extra space on line 112. You might also want to make sure that the line doesn't exceed 120 characters because the linter get's upset by that too!

Cheers for the contributions! 👍

Reply:
All done pushed. Please feel free to review.

Reply:
Oh no! [Four more](https://travis-ci.org/silverstripe/silverstripe-userforms/jobs/480164191#L656-L663) 'linting' errors!
Here's a tip ;)

use composer to install the linter locally on your development environment:
`composer install --dev squizlabs/php_codesniffer ^3`

Then you can run it over your code changes without having to push to GitHub:
`vendor/bin/phpcs --standard=vendor/silverstripe/userforms/phpcs.xml.dist vendor/silverstripe/userforms/code`

And here's the best bit - it can fix a lot of the issues itself! After running the `phpcs` line, follow it with `phpcbf`:
`vendor/bin/phpcbf --standard=vendor/silverstripe/userforms/phpcs.xml.dist vendor/silverstripe/userforms/code`

Note:
I've made assumptions here about your development installation. Please adjust commands accordingly if they're not correct (generally the paths).

Reply:
Thanks for your reply.